### PR TITLE
[NSTK-2164] Release 6.0.4 updated

### PR DESCRIPTION
**Pull request by haibinxie**
The only change between this PR and previous PR is the second commit, which change psctl to 1.0.3.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+#### Enhancements:
+
+* CSI 1.3 compliance.
+* Improve volume provisioning performance when topology is enabled.
+* Add default cockroach DB pod memory limit to 1GB.
+
+#### Bug Fixes:
+
+* Support HostName using FQDN.
```
Comment (dsupure) [inline CHANGELOG.md:17]: We don't have FQDN support.

File: pure-pso/values.yaml [hunk 2]
```diff
@@ -48,7 +48,22 @@ storagetopology:
 #       APIToken: "T-d4925090-c9bf-4033-8537-d24ee5669135"
 #       NFSEndPoint: "1.2.3.9"
 arrays:
-  FlashArrays: []
```
Comment (dsupure) [inline pure-pso/values.yaml:51]: Why we need these changes?

**Review by dsupure — COMMENTED** [review]
Nice, please address the comments I left.

**Review by david-enli — APPROVED** [review]
LGTM

**Review by dsupure — APPROVED** [review]
LGTM!

**Pull request merged**